fluentd/Gemfile

```diff
@@ -1,17 +1,18 @@
-gem 'fluentd', ENV['FLUENTD_VERSION']
-gem 'elasticsearch-transport', '7.5.0'
-gem 'elasticsearch-api', '7.5.0'
-gem 'elasticsearch', '7.5.0'
+source "https://rubygems.org"
+gem 'fluentd', ENV['FLUENTD_VERSION'], :source=>"https://rubygems.org"
+gem 'elasticsearch-transport'
+gem 'elasticsearch-api'
+gem 'elasticsearch'
 gem 'fluent-plugin-kubernetes_metadata_filter', '2.4.2'
-gem 'fluent-plugin-concat', '2.4.0'
-gem 'fluent-plugin-elasticsearch', '4.0.5'
-gem 'fluent-plugin-kafka', '0.13.0'
-gem 'fluent-plugin-multi-format-parser', '1.0.0'
-gem 'fluent-plugin-record-modifier', '2.1.0'
-gem 'fluent-plugin-rewrite-tag-filter', '2.2.0'
-gem 'fluent-plugin-systemd', '1.0.2'
-gem 'fluent-plugin-remote-syslog', '1.1'
-gem 'fluent-plugin-prometheus', '1.7.3'
-gem 'fluent-plugin-splunk-hec', '1.2.1'
+gem 'fluent-plugin-concat'
+gem 'fluent-plugin-elasticsearch'
+gem 'fluent-plugin-kafka'
+gem 'fluent-plugin-multi-format-parser'
+gem 'fluent-plugin-record-modifier'
+gem 'fluent-plugin-rewrite-tag-filter'
+gem 'fluent-plugin-systemd'
+gem 'fluent-plugin-remote-syslog'
+gem 'fluent-plugin-prometheus'
+gem 'fluent-plugin-splunk-hec'
```

Comment:
Why does this PR drop the version constraints initially introduced by #1858?

Reply:
IC, we rely on Gemfilelock.

Reply:
This was done in order to produce a Gemfile.lock that was consistent with 4.4  while testing.  The intention is to revert fluentd in https://github.com/openshift/origin-aggregated-logging/pull/1939 so I will look at applying those changes back in  #1939

Reply:
Actually ran into the following when defining versions which im not certain about the resolution:
```
Resolving dependencies...
Bundler could not find compatible versions for gem "activesupport":
  In Gemfile:
    fluent-plugin-splunk-hec (= 1.2.1) was resolved to 1.2.1, which depends on
      openid_connect (~> 1.1.8) was resolved to 1.1.8, which depends on
        activemodel was resolved to 5.2.4.3, which depends on
          activesupport (= 5.2.4.3)

    fluent-plugin-splunk-hec (= 1.2.1) was resolved to 1.2.1, which depends on
      activesupport (~> 5.2)
```